plugin/restore/mysql/mysql.go

```diff
@@ -615,49 +693,54 @@ func (r *Restore) getBinlogFilesMetaOnServer(ctx context.Context) ([]mysql.Binlo
 	}
 	defer rows.Close()
 
-	var binlogFiles []mysql.BinlogFile
+	var binlogFiles []BinlogFile
 	for rows.Next() {
-		var binlogFile mysql.BinlogFile
+		var binlogFile BinlogFile
 		var unused interface{}
 		if err := rows.Scan(&binlogFile.Name, &binlogFile.Size, &unused /*Encrypted column*/); err != nil {
 			return nil, err
 		}
 		binlogFiles = append(binlogFiles, binlogFile)
 	}
+
+	binlogFiles, err = sortBinlogFiles(binlogFiles)
+	if err != nil {
+		return nil, err
+	}
 	return binlogFiles, nil
 }
 
 // showLatestBinlogFile returns the metadata of latest binlog
```

Comment:
```suggestion
// getLatestBinlogFileMeta returns the metadata of latest binlog
```
Let's modify the comment